frontend/src/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -1,6 +1,34 @@
 import React, { Component } from 'react';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
 
 class CrossingDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: props.crossing.name,
+      description: props.crossing.description
+    };
+  }
+
+  updateCrossing(e) {
+    debugger;
+    this.props.updateCrossingMutation({
```

Comment:
This didn't persist changes for me on the full crossing list page. Maybe that's known, but this graphql fanciness is foreign to me, so I don't know any better than to ask

Reply:
In my testing the changes do persist, but require a refresh. This is a situation where we could call refetchqueries to get all of the crossings again, but I figured it'd be better to wait until we figured out a good solution for only refetching what we need.

I found a pattern that makes sense and implemented it on the crossing list page for status updates, moving forward it'll make sense to follow that pattern to propagate the crossing edits correctly.